src/utils.ts

```diff
@@ -41,6 +41,11 @@ export function isSPDXValid(license: string): boolean {
   }
 }
 
+// function to check if a value is not null or undefined
+export function isDefined<T>(value: T | null | undefined): value is T {
+  return value !== null && value !== undefined
+}
+
```

Comment:
This function is being used once in the whole codebase, we should inline the code instead of abstracting this to this sink file.

Reply:
True. Moved it inline - [0574926](https://github.com/actions/dependency-review-action/pull/423/commits/0574926a144976bce4cc4ead75a7733243974ec2)